package.json

```diff
@@ -19,7 +19,7 @@
     "start": "bin/start",
     "prepare": "npm run build",
     "build": "tsc",
-    "dev": "tsc-watch --project tsconfig.json --onSuccess 'node --inspect .'",
+    "dev": "tsc-watch --project tsconfig.json --onSuccess 'node --inspect-brk .'",
```

Comment:
`npm run dev` should be for running project, not waiting for brk point. 
`npm run debug` makes for sense for that 
IMHO

Reply:
Committed by accident while testing locally, but indeed useful as `debug`